Whiskey/en-US/about_Whiskey_DotNetPublish_Task.help.txt

```diff
@@ -0,0 +1,57 @@
+TOPIC
+    about_Whiskey_DotNetPublish_Task
+
+SUMMARY
+    Builds and publishes a .NET Core project and all its dependencies for deployment.
+
+DESCRIPTION
+    The `DotNetPublish` tasks runs the `dotnet publish` command which compiles the application, reads through its dependencies specified in the project file, and publishes the resulting set of files to a directory. The `DotNetPublish` task's output is ready for deployment to a hosting system. Pass a list of solutions files or .NET Core project files to the `Path` property. If no files are provided to `Path`, then the .NET Core SDK will search for any solution or project files in the working directory and build those. If the `dotnet publish` command returns a non-zero exit code (i.e. compilation error), the build will fail.

```

Comment:
Replace the first two sentence with "The `DotNetPublish` task runs the `dotnet publish` command which packages "the application and its dependencies into a folder for deployment". Your application won't run until you've published it. The `dotnet build` command doesn't create a runnable bin directory. Pass a list of solution..."